database/databasePostgresql/schemaEC2.sql

```diff
@@ -0,0 +1,77 @@
+DROP DATABASE IF EXISTS calendarteaa;
```

Comment:
You would probably want to add your CREATE INDEX statements at the bottom of the file, so it does it automatically after inserting data